pkg/runner/assets/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source ./lib.bash
+source ./env
+source ./steps.sh
+
+# shellcheck disable=SC1090
+step::run "step" "$@"
```

Comment:
```
INFO:   ====> running step::50::setup-validator-startup...
INFO:   ====> running step::run...
./lib.bash: line 96: $1: unbound variable
```

Step run is erroring after moving back to version in the lib. I'm reviewing but do not merge until fixed.